exercises/convertTemperature/src/handlers.js

```diff
@@ -0,0 +1,42 @@
+import {
+  SPACE,
+  TEMPERATURE_INPUT,
+  CONVERT_TEMPERATURE_CONTAINER,
+  MESSAGE_ERROR_NOT_INTEGER,
+} from './constants.js';
+import convertedTemperature from './utils/utils.js';
+
+const handlers = (event) => {
+  debugger; // each time the user changes the 'temperatures' input
+
+  const convertedTemperaturesContainer = document.getElementById(
+    CONVERT_TEMPERATURE_CONTAINER,
+  );
+
+  // Erase previous content
+  convertedTemperaturesContainer.innerHTML = '';
+
+  // Get the text input
+  const fahrenheitTextList = event.target.value;
+
+  let convertedTemp = convertedTemperaturesContainer.innerHTML;
+
+  // Validate it
+  if (fahrenheitTextList) {
+    var convertedTempTextList = convertedTemperature(fahrenheitTextList);
```

Comment:
Don't use `var`. It is not recommeded anymore. Use `let` or `const`.

```suggestion
    const convertedTempTextList = convertedTemperature(fahrenheitTextList);
```